frontend/src/components/ExpenseClaimSummary.vue

```diff
@@ -1,9 +1,9 @@
 <template>
 	<div class="flex flex-col w-full gap-5" v-if="summary.data">
 		<div class="text-lg text-gray-800 font-bold">Expense Claim Summary</div>
-		<div
-			class="flex flex-col gap-4 bg-white py-3 px-3.5 rounded-lg border-none"
-		>
+		<Autocomplete label="Payroll Period" class="w-full" placeholder="Select Payroll Period"
```

Comment:
![image](https://github.com/frappe/hrms/assets/24353136/7cf68eed-f180-4e80-b81e-d68722439b8b)

This autocomplete doesn't look good on a gray background. Move it somewhere inside the card like how the salary slip dashboard does it.